### PR TITLE
Update badges and documentation

### PR DESCRIPTION
**Pull request by damacus**
- Add badges from shields.io for Cookbook and Build status
- Add badge for pull reminders
- Add documentation folder for all resources

---
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/line/139)

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This is a big Pull Request.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by MarkGibbons — CHANGES_REQUESTED** [review]
The readme links to the resource documentation get 404 when clicked.  Should the links work? The links should probably work or be plain comments.

Fixed now.

File: README.md [hunk 1]
```diff
@@ -23,288 +25,21 @@ Quite often, the need arises to do line editing instead of managing an entire fi
...
-delete_lines      - Delete an item from a list
-add_to_list       - Add an item to a list
-delete_from_list  - Delete lines that match a pattern
-filter_lines      - Supply a proc or use a sample filter
+- [append_if_no_line](documentation/append_if_no_line.md) - Add a missing line
+- [replace_or_add](documentation/replace_or_add.md) - Replace a line that matches a pattern or add a missing line
+- [delete_lines](documentation/delete_lines.md) - Delete an item from a list
+- [add_to_list](documentation/add_to_list.md) - Add an item to a list
+- [delete_from_list](documentation/delete_from_list.md) - Delete lines that match a pattern
+- [filter_lines](documentation/filter_lines.md) - Supply a proc or use a sample filter
```
Comment (MarkGibbons) [inline README.md]: these need to be documentation/resources/resource.md

Comment (MarkGibbons) [reply README.md]: Fixed it.

**Review by MarkGibbons — APPROVED** [review]

**Review by MarkGibbons — COMMENTED** [review]
Changelog?  Up to you

**Comment by damacus** [comment]
Thanks for updating the links :) 

Fat fingered them!

**Comment by damacus** [comment]
Going to skip the changlog this time as there are no feature updates. Yolo

**Comment by MarkGibbons** [comment]
Didn't know you could do that in .md.  Very nice.

**Pull request merged**